scripts/screeps_rl_grafana.py

```
@@ -473,6 +524,27 @@ def exit_code_for_report(report: JsonObject) -> int:
     return 1
 
 
+def ensure_durable_container(command: list[str], container_name: str, restart_policy: str) -> int:
+    inspect = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)